src/components/MovieDetail/MovieDetail.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './MovieDetail.css';
+
+const MovieDetail = ({details}) => {
+  return (
+    <div className='movie-detail'>
+      <div className='movie-card'>
+        <img className='movie-card__image' src={details.backdrop_path} alt={details.title}/>
+        <section className='movie-card__details' >
+          <h2>{details.title}</h2>
+          <p>{details.average_rating}</p>
+          <p>{details.tagline}</p>
+          <p>{details.genres}</p>
+          <p>{details.overview}</p>
+          <p>{details.runtime}</p>
+          <p>{details.release_date}</p>
+        </section>
+      </div>
+    </div>
+  )
+}
```

Comment:
The rubric mentioned only passing down the props needed to children. I think that we are utilizing all the props passed down but we are also sending the entire object. I'm mostly just commenting about this so I remember this later on.

Reply:
As discussed just now, let's leave it for now but consider pruning once we get the API calls done.